Procfile

```diff
@@ -0,0 +1 @@
+web: node server
```

Comment:
We've been running the development server on Heroku. This fixes that. The frontend is built during the build process, and then served through the regular server.